opencti-platform/opencti-graphql/src/modules/workspace/workspace-domain.ts

```diff
@@ -135,3 +142,45 @@ export const workspaceEditContext = async (context: AuthContext, user: AuthUser,
   return storeLoadById(context, user, workspaceId, ENTITY_TYPE_WORKSPACE)
     .then((workspaceToReturn) => notify(BUS_TOPICS[ENTITY_TYPE_WORKSPACE].EDIT_TOPIC, workspaceToReturn, user));
 };
+
+function checkDashboardConfigurationImport(parsedData: any) {
+  if (parsedData.type !== 'dashboard') {
+    throw FunctionalError('Invalid type. Please import OpenCTI dashboard-type only', { reason: parsedData.type });
+  }
+
+  if (parsedData.version !== WORKSPACE_VERSION) {
+    throw FunctionalError(`Invalid version. Your workspace version must match the current one. Actual version : ${WORKSPACE_VERSION}`);
+  }
+}
+
+export const workspaceImport = async (context: AuthContext, user: AuthUser, file: Promise<FileHandle>) => {
+  const uploadedFile = await file;
+  const readStream = uploadedFile.createReadStream();
+  let fileContent = '';
+  await new Promise<void>((resolve, reject) => {
+    readStream.on('data', (chunk) => {
+      fileContent += chunk.toString();
+    });
+    readStream.on('end', () => {
+      resolve();
+    });
+    readStream.on('error', (error) => {
+      reject(error);
+    });
+  });
+  const parsedData = JSON.parse(fileContent);
+  checkDashboardConfigurationImport(parsedData);
+
+  const importWorkspaceCreation = await createEntity(context, user, parsedData, ENTITY_TYPE_WORKSPACE);
```

Comment:
which authorized members have this workspace ?

Reply:
This property is not managed in the duplication process.
Since it only includes : name, type, description, version, and manifest.

Reply:
Yes we can't copy the authorized members, but I think it should have the default authorized members we put when someone creates a dashboard, otherwise the dashboard is public, everyone can manage it. @Jipegien

Reply:
Yes. By default authorized_members. Need to be clearly documented in the official Doc.

Reply:
To make sure, I get it right : we want to add the user that importing the dashboard configuration as the only / first authorized member, @Jipegien.

Reply:
yes